emails/tests/utils_tests.py

```diff
@@ -91,6 +92,23 @@ def test_generate_relay_From_with_premium_user(self):
         )
         assert formatted_from_address == expected_formatted_from
 
+    def test_generate_relay_From_with_new_from_user(self):
+        free_user = make_free_test_user()
+        new_from_flag = Flag.objects.create(name="new_from_address")
+        new_from_flag.users.add(free_user)
+        original_from_address = '"foo bar" <foo@bar.com>'
+        formatted_from_address = generate_relay_From(
```

Comment:
I suggest an ``override_settings`` decorator to force ``RELAY_FROM_ADDRESS`` and ``NEW_RELAY_FROM_ADDRESS`` to different knows values

Failing locally before setting `NEW_RELAY_FROM_ADDRESS`: ``IndexError: string index out of range``